src/components/StudyRoom/StudyRoomSettings.js

```diff
@@ -0,0 +1,183 @@
+                            onChange={handleColorChange}
+                            value={roomData.color}
+                    >
+                        <MenuItem value={"#912338"}>
+                            <CircleIcon sx={{color: '#912338'}}/> Burgundy
+                        </MenuItem>
+                        <MenuItem value={"#DB0272"}>
+                            <CircleIcon sx={{color: '#DB0272'}}/> Magenta
+                        </MenuItem>
+                        <MenuItem value={"#DA3A16"}>
+                            <CircleIcon sx={{color: '#DA3A16'}}/> Orange
+                        </MenuItem>
+                        <MenuItem value={"#004085"}>
+                            <CircleIcon sx={{color: '#573996'}}/> Mauve
+                        </MenuItem>
+                        <MenuItem value={"#004085"}>
+                            <CircleIcon sx={{color: '#004085'}}/> Dark blue
+                        </MenuItem>
+                        <MenuItem value={"#0072A8"}>
+                            <CircleIcon sx={{color: '#0072A8'}}/> Blue
+                        </MenuItem>
+                        <MenuItem value={"#057D78"}>
+                            <CircleIcon sx={{color: '#057D78'}}/> Turquoise
+                        </MenuItem>
+                        <MenuItem value={"#508212"}>
+                            <CircleIcon sx={{color: '#508212'}}/> Green
+                        </MenuItem>
+                        <MenuItem value={"#E5A712"}>
+                            <CircleIcon sx={{color: '#E5A712'}}/> Yellow
+                        </MenuItem>
+                        <MenuItem value={"#CBB576"}>
+                            <CircleIcon sx={{color: '#CBB576'}}/> Gold
+                        </MenuItem>
+                    </Select>
+                    <Avatar sx={{bgcolor: roomData.color, width: 56, height: 56}}> {SetAvatarText(roomData.title)}
+
+                    </Avatar>
+                </Stack>
+                </div>
+                <div style={{width: '85vw', height: '50vh', marginTop: '10px'}}>
+                    <TextField
+                        fullWidth
+                        id='description'
+                        type='description'
+                        label="Description"
+                        variant='outlined'
+                        margin="normal"
+                        multiline
+                        maxRows={3}
+                        onChange={handleDescriptionChange}
+                        value={roomData.description}
+                    />
+                </div>
+                <Stack direction='row' spacing={7} marginTop={2}>
+                <PrimaryButton3 width={'41vw'} content="Update" onClick={handleUpdate} />
+                <PrimaryButton2 width={'41vw'} content="Delete" onClick={handleDelete}/>
+                </Stack>
+            </form>
+        </React.Fragment>
+    )
+    return(
+        updateRoom
```

Comment:
you dont need 3 lines to return this constant

Reply:
I find it more readable like that, but I put it on 1 line